### ISSUE TITLE
Console errors when Dialog/Overlay aren't provided with an onClose fn

### ISSUE THREAD

**Issue by mc-funk** [pluralsight/design-system#955]
'onClose' is expected by Overlay as a required prop, but Dialog and other components deliberately omit this so as to not have the close 'x' visible. 

This produces the following warning: 
```
Failed prop type: The prop `onClose` is marked as required in `Overlay`, but its value is `undefined`.
    in Overlay (created by ForwardRef)
```

Would it be possible to make this prop optional, or is this an issue of standardization where we need not to be omitting onClose? 

https://github.com/pluralsight/design-system/blob/edd33639d78e3e4dcf2b0b24f35ffe6959e44308/packages/actionmenu/src/react/overlay.js#L45

**Comment by EdwardIrby** [comment]
@mc-funk this should be resolved by #975. Let me know if it's not.

**Comment by mc-funk** [comment]
Many thanks! I've ticketed the work on our side to upgrade relevant packages and test this, so will circle back if we don't see what's expected.
